pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -125,6 +129,23 @@ func (s *SearchOptions) ComputeUserID() (int64, error) {
 	return strconv.ParseInt(id, 10, 64)
 }
 
+func (s *SearchOptions) HashString() (string, error) {
```

Comment:
Could we move it to cacheutils, seems like it's a better place for this code.

Reply:
Sure thing cd0b93b8ef023ef0bf644927d42246c9cd42bd39